### PR TITLE
Url redis

### PR DESCRIPTION
**Pull request by JackyRahman**
### Debio-1132
[https://blocksphere2020.atlassian.net/browse/DBIO-1132](https://blocksphere2020.atlassian.net/browse/DBIO-1132)

### [Back End] Create cache for daogenic maps
- create module cache redis
- change value of dbio exchange from API to Redis
- delete API set-dbio-balance

### CODE REVIEW

File: src/cache-redis/cache-redis.service.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { Injectable } from "@nestjs/common";
+import axios from "axios";
```
Comment (agustinustheo) [inline src/cache-redis/cache-redis.service.ts:2]: You already installed `@nestjs/axios` why do you need to import it here again. Inject the `HttpModule` and use it there.

File: src/cache-redis/cache-redis.module.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { Module } from "@nestjs/common";
+import { CacheRedisService } from "./cache-redis.service";
+
+@Module({
+	providers: [CacheRedisService],
+	exports: [CacheRedisService,]
+})
+export class CacheRedisModule {}
```
Comment (agustinustheo) [inline src/cache-redis/cache-redis.module.ts:8]: Don't name the module `CacheRedis` this doesn't use the cache at all, it calls an API that will get the conversion cache.

File: src/cache-redis/cache-redis.service.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+@Injectable()
+export class CacheRedisService {
+	constructor() {}
+	  async	getExchange() {
+      const res = await axios.get(`${process.env.REDIS_STORE_URL}/cache`, {
```
Comment (agustinustheo) [inline src/cache-redis/cache-redis.service.ts:7]: Use the `ProcessEnvModule` and the `ProcessEnvProxy` class. Accessing `process.env` directly removes the capability to unit test in the future.

**Review by agustinustheo — CHANGES_REQUESTED** [review]

File: src/cache-redis/cache-redis.module.ts (hunk 2 above)
Comment (agustinustheo) [reply src/cache-redis/cache-redis.module.ts:8]: Use another name like `DbioConversion` instead.

File: src/cache-redis/cache-redis.service.ts (hunk 3 above)
Comment (JackyRahman) [reply src/cache-redis/cache-redis.service.ts:7]: Okay Noted!
will enhance with debio-1180[https://blocksphere2020.atlassian.net/browse/DBIO-1180](https://blocksphere2020.atlassian.net/browse/DBIO-1180)

**Review by agustinustheo — APPROVED** [review]

**Pull request merged**